### PR TITLE
Feature/kien tt/t02_Display_the_first_100_jobs_done

### PR DESCRIPTION
**Pull request by truongtrungkien0999**
-Link deloy :https://kientt-job-board.surge.sh/
-Note
Use filter to get data display the first 100 jobs.
Unit tests, test some component complete.
-Result
image

### CODE REVIEW

**Comment by tyhvb** [comment]
Hi @truongtrungkien0999, the commit message should be more clear, it should be what you did

File: src/api/jobApi.ts [hunk 1]
```diff
@@ -2,8 +2,8 @@ import axios from "axios"
 import { Job } from "../state/constants/jobConstant";
 
 export const fetchJobsData = async (name: string) => {
-  // const { data } = await axios.get<Job[]>(`${process.env.REACT_APP_DATA_URL}/jobs`);
   const { data } = await axios.get<Job[]>(`https://6201d6f6b8735d00174cb605.mockapi.io/api/job`);
-  const jobs = data.filter((job: Job) => job.title.toLocaleLowerCase().includes(name.toLocaleLowerCase()));
+  const jobs = data.filter((job: Job) => Number(job.id)<=100);
```
Comment (tyhvb) [inline src/api/jobApi.ts:6]: I think this task's purpose is getting the first 100 jobs, not filtering jobs with id <=100, right

**Comment by tyhvb** [comment]
There is 1 failed test case after i ran `yarn test --watchAll`, could please fix it

File: src/api/jobApi.ts (hunk 1 above)
Comment (truongtrungkien0999) [reply src/api/jobApi.ts]: - Status : done 
- Note
I use array index instead of id to get first 100 jobs.

**Comment by truongtrungkien0999** [comment]
- Fix unit test file JobContentPreview.
- status : done.
- Note: fix search results in getbytext.

**Comment by truongtrungkien0999** [comment]
> There is 1 failed test case after i ran `yarn test --watchAll`, could u please fix it

Fix unit test file JobContentPreview.
status : done.
Note: fix search results in getbytext.

**Comment by tyhvb** [comment]
@truongtrungkien0999 No worries, but should write in English for familiar.
btw, should modify the PR title for more clear

File: src/api/jobApi.ts (hunk 1 above)
Comment (tyhvb) [reply src/api/jobApi.ts]: we can also use Array slice(), but this code is good to me, so resolve it

Comment (truongtrungkien0999) [reply src/api/jobApi.ts]: - Fix use array slice().
- status : done.
- please check again for me. thanks

File: package.json [hunk 2]
```diff
@@ -19,7 +19,8 @@
     "redux": "^4.1.2",
     "redux-devtools-extension": "^2.13.9",
     "redux-thunk": "^2.4.0",
-    "typescript": "~4.4.4"
+    "typescript": "~4.4.4",
+    "web-vitals": "^2.1.4"
```
Comment (tyhvb) [inline package.json:23]: Why do we need this package ?

Comment (truongtrungkien0999) [reply package.json]: i removed it from project. thanks

**Comment by tyhvb** [comment]
Yeah cool now @truongtrungkien0999 , And you also use this title for this PR: Feature/kien tt/t02 display the first 100 jobs

**Pull request merged**